Config.cmake.in

```diff
@@ -0,0 +1,7 @@
+include("${CMAKE_CURRENT_LIST_DIR}/@targets_export_name@.cmake")
```

Comment:
There's a huge discussion in #391 around whether cmake should be supported or something else.
One thing is for sure, this cmake solution will not be merged as it's unstructured and incomplete.